logos/tests/unit/dbutils/test_team_queries.py

```
@@ -0,0 +1,121 @@
+    session = MagicMock()
+    db.session = session
+    result, status = db.add_team_member(1, 5)
+    assert status == 200
+    assert result == {"result": "Member added"}
+    session.commit.assert_called_once()
+
+def test_add_team_member_as_owner():
+    db = DBManager.__new__(DBManager)
+    session = MagicMock()
+    db.session = session
+    result, status = db.add_team_member(1, 5, is_owner=True)
```

Ruff (0.15.12)

[warning] Unpacked variable `result` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)